### ISSUE TITLE
bouncer request module? 

### ISSUE THREAD

**Issue by itsheox3** [CatIRCBots/Granota#53]
Is there a way to have an ZNC request module?
Like for example if I have a command like `.request` then the bot will automatically make the ZNC user?
Is that possible?

**Comment by NeoMahler** [comment]
Yes, it's possible. However, I think that it shouldn't be included in Granota by default. Maybe I will create a repository called "Granota-extras" with it (when I create it).
